### PR TITLE
Document graph inverse semigroups

### PR DESCRIPTION
**Pull request by mtorpey**
This clears up all the "empty verb" errors.

### CODE REVIEW

File: doc/semigraph.xml [hunk 1]
```diff
@@ -71,9 +81,25 @@ v_2]]></Example>
 <ManSection>
   <Attr Name="Range" Arg="x" Label="for a graph inverse semigroup element"/>
   <Attr Name="Source" Arg="x" Label="for a graph inverse semigroup element"/>
-  <Returns></Returns>
+  <Returns>a graph inverse semigroup element</Returns>
```
Comment (wilfwilson) [inline doc/semigraph.xml]: This should be a sentence with a capital letter and full stop.

File: doc/semigraph.xml [hunk 2]
```diff
@@ -82,9 +108,31 @@ v_2]]></Example>
 <#GAPDoc Label="IsVertex">
 <ManSection>
   <Attr Name="IsVertex" Arg="x" Label="for a graph inverse semigroup element"/>
-  <Returns></Returns>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
   <Description>
+    If <A>x</A> is an element of a graph inverse semigroup
+    (i.e. it satisfies <Ref Filt = "IsGraphInverseSemigroupElement"/>) then this
```
Comment (wilfwilson) [inline doc/semigraph.xml]: Have a comma after the bracket:

`...), then this`

File: doc/semigraph.xml [hunk 3]
```diff
@@ -94,20 +142,44 @@ v_2]]></Example>
...
 
 <#GAPDoc Label="GraphOfGraphInverseSemigroup">
 <ManSection>
-  <Filt Name="GraphOfGraphInverseSemigroup" Arg="x"/>
-  <Returns></Returns>
+  <Attr Name="GraphOfGraphInverseSemigroup" Arg="S"/>
+  <Returns>A digraph.</Returns>
   <Description>
+    If <A>S</A> is a graph inverse semigroup (i.e. it satisfies
+    <Ref Filt="IsGraphInverseSemigroup"/>) then this attribute returns the
```
Comment (wilfwilson) [inline doc/semigraph.xml]: Another comma after the closing bracket. If you'd please be so kind.

File: doc/semigraph.xml (hunk 1 above)
Comment (mtorpey) [inline doc/semigraph.xml]: Fixed.

File: doc/semigraph.xml (hunk 3 above)
Comment (mtorpey) [inline doc/semigraph.xml]: Done both of these as well.

**Comment by mtorpey** [comment]
I've followed through on Wilf's suggestions.  Should be good to go.

**Comment by wilfwilson** [comment]
I'll merge once the tests pass, just to be 100% safe.

**Pull request merged**